s3/pkg/datastore/yig/meta/db/tidb/driver.go

```diff
@@ -0,0 +1,36 @@
+package tidb
+
+import (
+	"database/sql"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/opensds/multi-cloud/s3/pkg/datastore/yig/config"
+	"github.com/opensds/multi-cloud/s3/pkg/datastore/yig/meta/db/driver"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	TIDB_DRIVER_ID = "tidb"
+)
+
+type TidbDriver struct {
+}
+
+func (td *TidbDriver) OpenDB(dbCfg config.DatabaseConfig) (driver.DB, error) {
+	db := &Tidb{}
+	var err error
+	db.DB, err = sql.Open("mysql", dbCfg.DbUrl)
```

Comment:
do we use connection pooling ?

Reply:
@rhsakarpos the function call 'db.DB.SetMaxIdleConns' and 'db.DB.SetMaxOpenConns' will work for the connection pooling.